modules/oe_content_organisation/modules/oe_content_organisation_person/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenEuropa Organisation Person Reference
```

Comment:
Let's rename the module machine name as follows:
```
oe_content_organisation_person_reference
```

So we are consistent with the other module in the same directory, with similar purposes:

![image](https://user-images.githubusercontent.com/153362/117000749-d967a700-ace1-11eb-99d8-d5526b16c3a2.png)